plugins/destination/clickhouse/client/spec.go

```diff
@@ -47,6 +53,18 @@ func (s *Spec) SetDefaults() {
 	if s.Engine == nil {
 		s.Engine = queries.DefaultEngine()
 	}
+
+	if s.BatchSize == 0 {
+		s.BatchSize = writers.DefaultBatchSize
```

Comment:
Use plugin's own defaults (we're removing the exported writer values here for this reason)

Reply:
752b1df8ef92ab558baad9e4b57b425e4bfb4dbb